lib/options/sender_option_parser.rb

```diff
@@ -275,6 +277,16 @@ def initialize(args)
         @options.msg_subject = msg_subject
       end
 
+      # Anonymous producer
```

Comment:
`--anonymous` can be set as boolean option without true/false values, but setting one of two can be profit.

Reply:
dTests will always do `--anonymous false`. Or `true`. The way I did this, the argument is mandatory. And it works with dTests.

> but setting one of two can be profit.

I do not understand what you mean.

Reply:
javascript/nodejs has this as `--anonymous` without value. If present `= true`, otherwise `= false`.

This is simplier for handling, because we don't need to check true/True/yes/false/False/no.

> I do not understand what you mean.

But on the other hand, we can choose on CLI if we want true/false when testing manually, so we don't need to **delete** `--anonymous`, but only change true/false.

Reply:
If we want to unify options for all clients, then we want `--anonymous` without value.

Reply:
the deed is now donw